kuma/javascript/src/article.jsx

```diff
@@ -155,6 +152,9 @@ export default function Article({ document }: DocumentProps) {
 }
 
 function ArticleMetadata({ document }: DocumentProps) {
```

Comment:
This demonstrates how to do destructing instead of `(props: Props)`. Care to take a stab?